solutions/usaco-736.mdx

```diff
@@ -188,3 +188,75 @@ public class BovineGenomics {
 </LanguageSection>
 
 
+## Solution 3 - Unordered Map
+
+<LanguageSection>
+
+<CPPSection>
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(M * N)$
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+int n, m, ans = 0;
+vector<string> gens;
+
+// Use `unordered_map` to achieve O(1) time complexity in queries
+unordered_map<char, int> spotties;
+
+int main()
+{
+  ios::sync_with_stdio(false);
```

Comment:
If the code works without these then remove them
Also the style guide explicitly mentions that `cout.tie(0)` is not allowed
![image](https://user-images.githubusercontent.com/67491820/140461554-7f79e078-b4d4-465c-a006-c4aa8e8d3a79.png)